R/ci.R

```diff
@@ -124,43 +219,64 @@ use_coveralls_badge <- function() {
   use_badge("Coverage status", url, img)
 }
 
-#' @section `use_appveyor()`:
-#' Adds a basic `appveyor.yml` to the top-level directory of a package. This is
-#' a configuration file for the [AppVeyor](https://www.appveyor.com) continuous
-#' integration service for Windows.
+#' Setup deployment for Travis CI
+#'
+#' Creates a public-private key pair,
+#' adds the public key to the GitHub repository via [github_add_key()],
+#' and stores the private key as an encrypted environment variable in Travis CI
+#' via [travis_set_var()],
+#' possibly in a different repository.
+#' The \pkg{tic} companion package contains facilities for installing such a key
+#' during a Travis CI build.
+#'
+#' @importFrom travis travis_set_var
+#' @importFrom travis github_add_key
+#' @importFrom tic get_public_key
+#' @importFrom tic encode_private_key
+#' @importFrom openssl rsa_keygen
+#' @param path `[string]`\cr
+#'   The path to a GitHub-enabled Git repository (or a subdirectory thereof).
+#' @param info `[list]`\cr
+#'   GitHub information for the repository, by default obtained through
+#'   [github_info()].
+#' @param repo `[string|numeric]`\cr
+#'   The GitHub repo slug, by default obtained through [github_repo()].
+#'   Alternatively, the Travis CI repo ID, e.g. obtained through `travis_repo_id()`.
+#' @param travis_repo `[string]`\cr
+#'   The Travis CI repository to add the private key to, default: `repo`
+#'   (the GitHub repo to which the public deploy key is added).
 #' @export
-#' @rdname ci
-use_appveyor <- function(browse = interactive()) {
-  check_uses_github()
+use_travis_deploy <- function(path = ".", info = travis:::github_info(path),
```

Comment:
`github_repo()` and `github_repo()` are both exported, not sure we need `info` at all.

Reply:
I removed `info`.